packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "04085c80c2c84ea6b8375042d61797f7f46138fd",
-    "ref_origin" : "dcos-mesos-master-f179400b"
+    "ref": "9d1cb5ecba1a7921b067ad94092c3663adf19b24",
+    "ref_origin" : "dcos-mesos-1.2.0-rc1"
```

Comment:
Is this really the branch you pushed to on mesosphere/mesos? I tried to search for https://github.com/mesosphere/mesos/tree/dcos-mesos-1.2.0-rc1 and accidentally created a branch synced to apache:master.. :(
The SHAs in https://github.com/mesosphere/mesos/compare/04085c80c2c84ea6b8375042d61797f7f46138fd...9d1cb5ecba1a7921b067ad94092c3663adf19b24 stil check out, so that's good.